guides-and-policies/policy-source-control.md

```diff
@@ -43,9 +44,12 @@ At Arcadia Science, repositories should follow basic guidelines.
-- **Composition of Pull Requests:** Each pull request may contain one or more code changes. In keeping with good source control practice, each change or commit should contain all changes necessary for a particular fix or update. In addition, each pull request should relate to no more than one functional area in the code base you are updating. Keeping the pull request focused to one area makes it easier for your reviewers to provide thoughtful feedback.
+- **Composition of Pull Requests:** Each pull request may contain one or more code changes as long as they focus on a specific task. In keeping with good source control practice, each change or commit should contain all changes necessary for a particular fix or update. In addition, each pull request should relate to no more than one functional area in the code base you are updating. Keeping the pull request focused to one area will most likely keep them short. Short pull requests make it easier for your reviewers to provide thoughtful and quick feedback. Quick feedback means you can quickly address concerns and get your pull request merged. In general, shorter and more focused pull requests are better than longer ones.
 - **Reviewing Pull Requests:** We expect that all team members will participate in review of pull requests. If you get named by the submitter, it's courteous to review the request. People can get busy or miss the review submission. It’s the responsibility of the person seeking review to pursue the reviewer to make sure code changes are committed in a timely manner. While all code-related discussion should occur publicly on GitHub, it is appropriate to nudge people about reviews or check about their timeline using DMs in slack.
   - Potential suggested code review checklist: [https://github.com/greenelab/onboarding/blob/master/extras/code_review_checklist.md](https://github.com/greenelab/onboarding/blob/master/extras/code_review_checklist.md)
+- **Merging Pull Requests:** Once your code is reviewed and pull request is approved, it's time to merge your changes. There are 3 methods of merging a PR via git: **Merge** will create a merge commit that joins two branches together. **Rebase** rewrites history on top of a branch to create a linear history. **Squash and merge** creates a single new squashed commit that encompasses all commits in the feature branch. At Arcadia Science, we use the first option and merge pull requests by creating a merge commit. You can enforce this policy by updating your repository settings to look like this:
```

Comment:
Ok! but why does this matter? so the release notes are more complete with the full commit history?